centovacast.php

```diff
@@ -1826,13 +1831,37 @@ private function getFieldsFromInput(array $vars, $package)
         ];
 
         $fields = array_merge((array) $package->meta, $fields);
-
-        if ($fields['diskquota'] == 0) {
-            $fields['diskquota'] = 'unlimited';
-        }
-        if ($fields['transferlimit'] == 0) {
-            $fields['transferlimit'] = 'unlimited';
-        }
+		
+		//configoptions (Configurable Options)
+		$fieldsConfigOptions = array_merge((array) $package->meta, $fields['configoptions']);
+		
+		
+		//If centovacast_diskquota configoptions is specified
+		if (isset($fieldsConfigOptions['diskquota'])) {
+			if ($fieldsConfigOptions['diskquota'] == 0) {
+				$fields['diskquota'] = 'unlimited';
+			}
+			else {
+				$fields['diskquota'] = $fieldsConfigOptions['diskquota'];
+			}
+		}
+		elseif ($fields['diskquota'] == 0) {
+			$fields['diskquota'] = 'unlimited';
+		}
+		
+		
+		//If centovacast_transferlimit configoptions is specified
```

Comment:
This code is duplicate of 1839-1850.  Try to keep the code DRY.  In this case you could create a loop:
`fields = quota, transfer`
`foreach fields as  field`
`    run code similar to lines 1839-1850`